lib/apiservers/engine/backends/backends.go

```diff
@@ -212,8 +196,18 @@ func ProductVersion() string {
 	return productVersion
 }
 
+// VchConfig gets the VIC appliance VM's configspec
 func VchConfig() *config.VirtualContainerHostConfigSpec {
-	return vchConfig
+	var vchConfig config.VirtualContainerHostConfigSpec
```

Comment:
After chatting with @hickeng, it appears we are not going to get config pushed to us.  Instead we're going to pull more restrictive config from somewhere else so I reverted the code back to caching the VCH config.